src/components/viewer.js

```diff
@@ -20,10 +20,26 @@ const renderChildren = (nodes, isListItem) =>
     }
     // Text node
     if (typeof node === 'string') {
-      return node;
+      const line = node.replace(/\n/g, "");
```

Comment:
yes it well, but I'm nesting each line within a `span` with `display: block`, so that will force new lines without `\n`. I was getting strange side-effects if I left those line breaks in there.